libsrc/jsonserver/JsonClientConnection.cpp

```diff
@@ -486,9 +486,9 @@ void JsonClientConnection::handleCreateEffectCommand(const QJsonObject& message,
 				
 				if (effectArray.size() > 0)
 				{
-					if (message["name"].toString().trimmed().isEmpty())
+					if (message["name"].toString().trimmed().isEmpty() || message["name"].toString().trimmed().cbegin() == ".")
```

Comment:
Ah! Next time i try the js way instead something different ;)
Is there somewhere a good overview with good examples for beginners?

For js i found the mozilla developer network. Very good to understand the basics and how they behave, list of possibe functions etc. The QT5 doc feels a step too high :)